packages/nimble-components/src/table/specs/table-columns-hld.md

```diff
@@ -238,7 +301,7 @@ const isPositive = (value: number): bool => {
     return value >= 0;
 }
 
-public class TableColumnPositiveNegativeNumber extends TableColumn<TableColumnPositiveNegativeNumberCellData> {
+public class TableColumnPositiveNegativeNumber extends TableColumn {
```

Comment:
Reminder to update the implementation below to use the new custom element tag for its `cellViewTag` override.

Reply:
Updated